assets/src/edit-story/components/library/panes/shapes/shapesPane.js

```diff
@@ -72,7 +72,7 @@ function ShapesPane(props) {
               key={mask.type}
               onClick={() => {
                 insertElement('shape', {
-                  backgroundColor: '#333',
+                  resource: { color: '#333' },
```

Comment:
Let's remove `resource` use for this. IMHO, resources should be always invariant, not something that the editor changes.